src/cordova/get_test_parameters.mjs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import minimist from 'minimist';
+import os from 'os';
+
+const VALID_PLATFORMS = ['linux', 'windows', 'ios', 'macos', 'android', 'browser'];
+
+const DEFAULT_IOS_DEVICE_MODEL = 'iPhone 14';
+const DEFAULT_MACOS_ARCH = 'x86_64';
+
+/*
+  Inputs:
+  => platform: the list of action arguments passed in
+
+  Outputs:
+  => an object containing the specificed platform and buildMode.
+*/
+export function getTestParameters(buildParameters) {
```

Comment:
I'm finding it harder to understand the test because of this function. Can we have this logic in the test action instead?

The fact that this is mixing non-cordova platforms and things not needed for android makes it more confusing.

Also, it's not feasible to have one set of test parameters for all test actions. The parameters are action-dependent. Should live in the action